assets/src/application/ui/forms/espressoForm/renderers/FieldRenderer.tsx

```diff
@@ -45,7 +51,7 @@ const FieldRenderer: React.FC<FieldRendererProps> = (props) => {
 		>
 			<>
 				{before}
-				<MappedField {...rest} />
+				<MappedField aria-label={label} aria-describedby={tooltipKey} {...rest} />
```

Comment:
not in alphabetical order :sob: 
![alphabet-animal](https://user-images.githubusercontent.com/1477453/75972940-06051080-5edd-11ea-958f-04ec2d5675f2.jpg)

Reply:
LOLZ